tests/cache.py

```diff
@@ -22,8 +23,21 @@ def function(self, items):
     @inlineCallbacks
     def testCache(self):
         items = []
+        
         yield self.function(items)
         yield self.function(items, cache_key=1, cache_ttl=10)
         self.assertEqual(len(items), 2)
         yield self.function(items, cache_key=1, cache_ttl=10)
+        yield self.function(items, cache_key=1, cache_ttl=10)
+        self.assertEqual(len(items), 2)
+        
+        # Test CACHE autocleaning
         self.assertEqual(len(items), 2)
+        
+        _tmp = CACHE.max_live_time
```

Comment:
Sorry, that I'm too picky, but:
1) This is ugly.
2) Why we need max_live_time property at all? We can remove cache key right after it's TTL expired.
It could be different for different keys.
3) Test doesn't check cleanup. To check cleanup we some refactoring:

- add new test method, not extend existing
- name it right, without comments
- test CacheClass itself, without @cache decorator
- add now parameter to cleanup method (to not waiting TTL passes and shift now time)
  reactor.seconds() already calculated in wrapper method
- test must check that keys is really cleaned out from dictionary